MySaver.WebApi/Models/Store.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace WebApiTest.Models;
+
+public class Store
+{
+    public string Name { get; set; }
+
+    
+    [Key] 
+    public string Address { get; set; }
+
+    //[AllowNull]
+    //public Dictionary<DayOfWeek, string> Schedule { get; set; }
+
+    public string DefaultSchedule { get; set; }
+
+    /*
```

Comment:
You should probably delete this and other comments since they don't really help understand the code. Also you haven't updated Store model in MySaver project to reflect these changes.

Reply:
Yea, nice catch! The new commit should work now.